igvm/settings.py

```diff
@@ -3,6 +3,10 @@
 Copyright (c) 2018, InnoGames GmbH
 """
 
+import logging
+
+logging.getLogger('paramiko').setLevel(logging.WARNING)
+
```

Comment:
But don't I want exactly that? I added this because for DRBD migration displaying progress bar was impossible because every read of `/proc/drbd` was producing multiple lines of output. Furthermore, I would call this "debug output", even though its level was "info".

Reply:
You are not changing the level for igvm though, but for every usage of the global paramiko logger anywhere. Daniel made paramiko logging configurable in iggop and you will change our logging configuration inadvertently. The python docs also explicitly ask library devs not to attach loggers, see the Note at the end of https://docs.python.org/2/howto/logging.html#configuring-logging-for-a-library

Reply:
Let me check if I can change it just for the progress bar displaying. But no matter if it is possible or not, I consider this to be debug output, not any kind of "info". But well, that is Paramiko, I don't expect much better code quality.